OpencartTest/src/com/opencart/tools/Randomizer.java

```diff
@@ -9,8 +9,11 @@
     private static final String NUMBER = "0123456789";
 
     public static String generateRandomString(int length) {
```

Comment:
awesome, but in real life use apache.commons.lang lib and it's RandomStringUtils, no need for bicycle development 
OR use default Random class for digits and convert to string

Reply:
done!